README.md

```diff
@@ -58,4 +58,6 @@ Open `launch.json` (via the gear wheel in the run config dropdown in the debug t
     "juliaEnv": ".",
     "args": []
 }
-```
\ No newline at end of file
+```
+
+The "args" should contain the path to the .json input file 
```

Comment:
The given debugger config is for running all tests. Debugging a specific simulation run works similarly and requires the input path in the `args` argument, but perhaps this should be explained in more detail in a seperate paragraph as it is not correct here.